docs/sources/operators-guide/run-production-environment/production-tips/index.md

```diff
@@ -113,12 +111,11 @@ For more information about required disk space, refer to [Compactor disk utiliza
 
 ### Ensure Memcached is properly scaled
 
-We recommend ensuring Memcached evictions happen infrequently.
+Ensure Memcached evictions happen infrequently.
 Grafana Mimir query performances might be negatively affected if your Memcached cluster evicts items frequently.
```

Comment:
```suggestion
Grafana Mimir query performances might be negatively affected if your Memcached cluster evicts items frequently.
To prevent this, make sure that Memcached evictions happen infrequently.
```